common/models/create_deploy_request.go

```diff
@@ -1,16 +1,22 @@
 package models
 
-import "fmt"
+import (
+	"fmt"
+)
 
 type CreateDeployRequest struct {
 	DeployName string `json:"deploy_name"`
-	Dockerrun  []byte `json:"dockerrun"`
+	DeployFile []byte `json:"deploy_file"`
 }
 
 func (c CreateDeployRequest) Validate() error {
 	if c.DeployName == "" {
 		return fmt.Errorf("Deploy Name is required")
 	}
 
+	if c.DeployFile == nil {
```

Comment:
[nit] checking `if len(c.DeployFile) == 0` is safe even if a slice is `nil`, and is a little bit more robust.

Reply:
✅